app/model.js

```diff
@@ -22,4 +22,28 @@ exports.selectAllArticles = ()=>{
 
 exports.selectSpecifiedComments = (id)=>{
     return db.query(`SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`,[id])
-}
\ No newline at end of file
+}
+
+exports.updateComments = (comment,id)=>{
+
+    const {body} = comment
+    const {article_id} = id
+    const author = comment.username
+ 
+    if(body===undefined||article_id===undefined||author===undefined){
```

Comment:
a more readable way to write this would be `if (!body || !article_id || !author)`